pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -28,6 +28,43 @@
 INV_SQRT2 = 1 / qml.math.sqrt(2)
 
 
+class Barrier(Operation):
```

Comment:
Might be better to stick this lower in the file, so the most important gates, Pauli's and Hadamard, are up top.